### ISSUE TITLE
Using the pretrained models 

### ISSUE THREAD

**Issue by SJabin** [rakshithShetty/A4NT-author-masking#4]
Hi, 

How to evaluate the pre-trained models to reproduce the results from the paper?
Much appreciated.
